javascripts/main.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready(function() {
+  function searchMovie(movieName){
+    $.ajax({
+      type: 'GET',
+      url: `https://api.themoviedb.org/3/search/company?api_key=380b0e09e540c1e5680debdf9ad794f3&page=1&query=${movieName}`,
+      success: function(data) {
+
+         movies.innerHTML=""
```

Comment:
Where is this 'movies' object or DOM item defined? Does this code work? Did you intend to write this:?
```
let $movies = $('.movies') // or $('#movies')
$movies.innerHTML = ""
```